### ISSUE TITLE
Typos discovered by codespell

### ISSUE THREAD

**Issue by cclauss** [GEM-benchmark/NL-Augmenter#69]
codespell --ignore-words-list="fro,ist,oder"

```
./dataset.py:122: relavent ==> relevant
./dataset.py:143: hierachy ==> hierarchy
./notebooks/Write_a_sample_transformation.ipynb:1442: tht ==> the, that
./notebooks/Write_a_sample_transformation.ipynb:1718: exisiting ==> existing
./evaluation/evaluate_text_generation.py:84: upto ==> up to
./transformations/change_two_way_ne/README.md:11: implemetation ==> implementation
```

**Comment by kaustubhdhole** [comment]
Thank you very much @cclauss for pointing them out!

**Comment by AbinayaM02** [comment]
Closing the issue, since it's fixed in #118.
